### PR TITLE
Gradle updates

### PR DESCRIPTION
**Pull request by CodeCombiner**

### CODE REVIEW

File: android/manifest.xml [hunk 1]
```diff
@@ -49,10 +49,12 @@
 <!--START_PLUGINS_APPLICATION-->
 <!-- Onesignal -->
 <meta-data android:name="com.onesignal.BadgeCount" android:value="DISABLE" />
+
 <meta-data android:name="googleProjectNo" android:value="" />
 <meta-data android:name="onesignalAppID" android:value="" />
+<!--
 <meta-data android:name="com.google.android.gms.version"
-           android:value="@integer/google_play_services_version" />
+           android:value="@integer/google_play_services_version" /> -->
```
Comment (rampr) [inline android/manifest.xml]: Can be removed if it is not used

File: android/manifest.xml [hunk 2]
```diff
@@ -67,15 +69,6 @@
   </intent-filter>
 </receiver>
 
-<receiver android:name="com.onesignal.GcmBroadcastReceiver"
```
Comment (rampr) [inline android/manifest.xml:70]: Is it okay to remove these ?
Has this plugin migrated to FCM ?

Comment (CodeCombiner) [reply android/manifest.xml:70]: Onesignal has not moved to FCM still. We should keep play services settings

Comment (CodeCombiner) [reply android/manifest.xml:70]: Above settings are not required as I see in OneSignal manuals:
https://documentation.onesignal.com/docs/android-sdk-setup
I will remove them and test how it works with OneSignal.

Comment (rampr) [reply android/manifest.xml:70]: cool thanks

File: android/manifest.xml (hunk 1 above)
Comment (CodeCombiner) [reply android/manifest.xml]: Removed

**Review by rampr — APPROVED** [review]

**Review by vivekb91 — APPROVED** [review]

**Pull request merged**